### PR TITLE
Chris work page

### PR DESCRIPTION
**Pull request by chrisjzamora8**
Updated the work section - appreciate recommendations in case for future projects that are implemented on this page.

### CODE REVIEW

File: work.html [hunk 1]
```diff
@@ -37,11 +37,12 @@ <h1>Just some of our latest projects.</h1>
                 <ul id="portfolio-contant-active">
                     <li class="mix Branding">
                       <a href="#">
-                        <img src="{{ site.baseurl }}/img/portfolio/work1.jpg" alt="">
+                        <img src="img/Gazette.png" alt="">
```
Comment (fedebassetti) [inline work.html]: `img src="{{ site.baseurl }}/img/portfolio/work1.jpg" alt="">`
- [ ] Need to use the baseurl variable to generate a full path for the image to be found.
- [ ] the gazette file should go into the portfolio folder, or we can make a different one if it makes more sense
- [ ] keep the filename lowercase

File: work.html [hunk 2]
```diff
@@ -37,11 +37,12 @@ <h1>Just some of our latest projects.</h1>
...
                         <div class="overly">
                           <div class="position-center">
-                            <h2>Tesla Motors</h2>
-                            <p>Labore et dolore magna aliqua. Ut enim ad </p>
+                            <h2>OM&IS Gazette</h2>
+                            <p>The OM&IS Gazette was inspired by Russ Devereaux. It contains 
+                              News and updates from the NIU OM&IS department  </p>
```
Comment (fedebassetti) [inline work.html]: - [ ] grab the text blurb from the `about` page of the omis gazette and see if it fits.

File: work.html [hunk 3]
```diff
@@ -37,11 +37,12 @@ <h1>Just some of our latest projects.</h1>
                 <ul id="portfolio-contant-active">
                     <li class="mix Branding">
                       <a href="#">
```
Comment (fedebassetti) [inline work.html]: - [ ] add omis gazette link `https://inspiredlab.github.io/omisgazette/`

File: work.html [hunk 4]
```diff
@@ -36,12 +36,13 @@ <h1>Just some of our latest projects.</h1>
...
-                        <img src="{{ site.baseurl }}/img/portfolio/work1.jpg" alt="">
+                      <a href="https://inspiredlab.github.io/omisgazette/">
+                        <img src="{{ site.baseurl }}/img/Gazette.png" alt="OM&IS Gazette Logo">
                         <div class="overly">
                           <div class="position-center">
-                            <h2>Tesla Motors</h2>
-                            <p>Labore et dolore magna aliqua. Ut enim ad </p>
+                            <h2>OM&IS Gazette</h2>
+                            <p>The OM&IS Gazette was inspired by Russ Devereaux. It contains 
+                              news and updates from the NIU OM&IS department  </p>
```
Comment (cwalters126) [inline work.html]: small n for news here

File: work.html (hunk 3 above)
Comment (fedebassetti) [reply work.html]: updated

File: work.html (hunk 4 above)
Comment (fedebassetti) [reply work.html]: resolving @cwalters126 made the change

File: work.html (hunk 2 above)
Comment (fedebassetti) [reply work.html]: resolving @cwalters126 ok

File: work.html (hunk 1 above)
Comment (fedebassetti) [reply work.html]: all done resolving

**Comment by fedebassetti** [comment]
@chrisjzamora8 once @cwalters126 approves you can merge the changes.

**Pull request merged**